tensornetwork/backends/shell/shell_backend_test.py

```diff
@@ -0,0 +1,56 @@
+# Copyright 2019 The TensorNetwork Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import numpy as np
+import pytest
+from tensornetwork import tensornetwork_test
+
+
+def get_shape(a):
+  if isinstance(a, int) or isinstance(a, float):
+    return tuple()
+  else:
+    return a.shape
+
+def assertShapesEqual(a, b, rtol=1e-8):
+  assert get_shape(a) == get_shape(b)
+
+@pytest.fixture(
+  name="backend", params=["shell"])
+def backend_fixure(request):
+    return request.param
+
+# Override np.testing to check only shapes and not values

```

Comment:
I'd rather you'd explictly mark the tests as a skip for the shell backend rather than doing this weird `set(dir(...))` hack that will become difficult to maintain.

Reply:
Sorry, just saw this comment. The issue is that I don't want to skip the tests. I want to run all the `tensornetwork_test.py` tests but instead of doing `np.testing.assert_allclose` only test that the shapes agree. The hack seems to screw up other tests so it is probably a bad idea...